bundles/test/org.eclipse.smarthome.test/build.properties

```diff
@@ -2,4 +2,6 @@ output.. = target/classes/
 bin.includes = META-INF/,\
                .,\
                NOTICE
-source.. = src/main/groovy/
+source.. = src/main/groovy/,\
+           src/test/java/,\
+           src/main/java/
```

Comment:
This line somehow forces tycho to choose the maven compiler which only picks up the .java files. So OSGiTest.groovy is missing which causes the maven build error. Eclipse IDE is capable of building the project because it does not rely on `build.properties`.

My suggestion is to refactor the two remaining groovy files to java files. In the case of the OSGiTest the JavaOSGiTest should be API compatible so it can be duplicated in the first step. The AsyncResultWrapper is already nearly a valid Java class.

In a second step we could unite the OSGiTest and the JavaOSGiTest back into a single file.

Reply:
In case this is not doable right now, please revert this lines to let the build pass.